### ISSUE TITLE
TRYING TO COMPILE FROM 2 DAYS

### ISSUE THREAD

**Issue by nft30001** [phrutis/Rotor-Cuda#14]
Hi bro i am sorry to bother you but please need small help from your side . i know you already have all VS 2019 setup and can you compile this for me as i am trying from 2 days and it is not working at all. just please edit following and compile for windows.

```
`if (rKey > 0) {
			if ((count - lastrKey) > (10000 * rKey)) {
				// rKey request
				rKeyRequest(params);
				lastrKey = count;
				rKeyCount++;
				rKeyCount2 += rKeyCount;`
```

 i need this with 10000 value ~ Thanks

**Comment by phrutis** [comment]
Reload every 10,000? What for?
This is not suitable for a GPU!
This is not enough for CPUs.
If you need Full random try mode 7 at LostCoins

**Comment by nft30001** [comment]
this is only for my test please , can you help in this ? i am not asking official ~ just asking for my testing purpose

**Comment by phrutis** [comment]
In 2 hours I will be at home, I will collect it for you. Have you figured out the last question where you did not find it in the random house?

**Comment by phrutis** [comment]
Can I make a parameter for you instead of 10,000? 
For example -m 1-1000000000 How it will be more convenient to choose what you want

**Comment by nft30001** [comment]
> In 2 hours I will be at home, I will collect it for you. Have you figured out the last question where you did not find it in the random house?
  
yes bro thank to you ~ you explain well so i change grid size to 256,256 and this was ok after that.

**Comment by phrutis** [comment]
I don't understand why you need this, [download](https://github.com/phrutis/Rotor-Cuda/blob/main/Others/Rotor-Cuda-test.exe), test...
For test only. Use **-r** from (min) 1 to 18,446,744,073,709,551,615 (max)
Example: ```-r 10000```
```
if (rKey > 0) {
			if ((count - lastrKey) > rKey) {
				// rKey request
				rKeyRequest(params);
				lastrKey = count;
				rKeyCount++;
				rKeyCount2 += rKeyCount;
			}
		}
```

**Comment by phrutis** [comment]
Work?

**Comment by studyroom1** [comment]
yes bro  i will check on my pc and will tell you . thanks

**Comment by nft30001** [comment]
> I don't understand why you need this, [download](https://github.com/phrutis/Rotor-Cuda/blob/main/Others/Rotor-Cuda-test.exe), test... For test only. Use **-r** from (min) 1 to 18,446,744,073,709,551,615 (max) Example: `-r 10000`
> 
> ```
> if (rKey > 0) {
> 			if ((count - lastrKey) > rKey) {
> 				// rKey request
> 				rKeyRequest(params);
> 				lastrKey = count;
> 				rKeyCount++;
> 				rKeyCount2 += rKeyCount;
> 			}
> 		}
> ```

hi bro not this one actually i want above codes like this one please 

```
if (rKey > 0) {
			if ((count - lastrKey) > (10000 * rKey)) {
				// rKey request
				rKeyRequest(params);
				lastrKey = count;
				rKeyCount++;
				rKeyCount2 += rKeyCount
```

**Comment by phrutis** [comment]
Use -r 10000
